lib/debuglet.js

```diff
@@ -215,7 +217,21 @@ Debuglet.prototype.scheduleBreakpointFetch_ = function(seconds) {
 
         default:
           that.logger_.info('\t' + response.statusCode + ' completed.');
-          that.updateActiveBreakpoints_(body.breakpoints || []);
+          var bps = (body.breakpoints || []).filter(function(bp) {
+            if (bp.action !== 'CAPTURE') {
```

Comment:
Let's verify that `bp.action` is always/presently-being provided by the API. The proto seems to suggest this is optional.